### PR TITLE
Feature login auth

### PR DESCRIPTION
**Pull request by cdsandoval**

### CODE REVIEW

File: src/action-hook.jsx [hunk 1]
```diff
@@ -5,9 +5,22 @@ import {
   decreaseQuantity,
   reset,
   addCart,
-  removeCart
+  removeCart,
```
Comment (byteconde) [inline src/action-hook.jsx]: Well done Carlos, we're going to need those actions to complete our project

File: src/action.jsx [hunk 2]
```diff
@@ -1,3 +1,7 @@
+const API_LOGIN_URL = "http://localhost:3000/api/login";
+const API_LOGOUT_URL = "http://localhost:3000/api/logout";
+const API_RESTAURANT_URL = "http://localhost:3000/api/restaurants";
+
```
Comment (byteconde) [inline src/action.jsx]: Probably you could wrap localhost:3000 in const

File: src/action-hook.jsx (hunk 1 above)
Comment (cdsandoval) [reply src/action-hook.jsx]: Thank you. Hope this works.

File: src/action.jsx (hunk 2 above)
Comment (cdsandoval) [reply src/action.jsx]: Hmm It could a good option but I am doing this to got a better notion of where I am fetching with the direct url.

**Pull request merged**